### PR TITLE
Add the current schema as initial migration script

### PR DESCRIPTION
**Pull request by ghostwriternr**

### CODE REVIEW

**Comment by ghostwriternr** [comment]
Booo @pep8speaks

File: c3po/job/utils/facebook.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def refresh_access_token():
         dotenv.get_key(dotenvfile, "FB_LONG_ACCESS_TOKEN")
         warns = filter(lambda i: issubclass(i.category, UserWarning), warns)
         if warns:
-            request_url = FB_URL + "oauth/access_token"
+            request_url = FB_BASE_URL + "oauth/access_token"
```
Comment (thescriptninja) [inline c3po/job/utils/facebook.py:52]: Shouldn't this remain `FB_URL` ? 🤔

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,6 +55,7 @@ jobs:
           command: |
             . env/bin/activate
             ./bin/sanity.sh 
+            alembic upgrade head
```
Comment (mukul-mehta) [inline .circleci/config.yml:58]: Won't this run migrations on the `CircleCI` server? I believe we need to add it to our `deploy.sh` script to make it run on the server where it is deployed.

File: c3po/job/utils/facebook.py (hunk 1 above)
Comment (ghostwriternr) [reply c3po/job/utils/facebook.py:52]: I thought so too. But I was running into errors when I ran the script.
Checked the documentation: https://developers.facebook.com/docs/facebook-login/access-tokens#generating-an-app-access-token and found out it is supposed to just be `https://graph.facebook.com/oauth/access_token` for refreshing tokens.

File: .circleci/config.yml (hunk 2 above)
Comment (ghostwriternr) [reply .circleci/config.yml:58]: You're absolutely right. I missed reading the config thoroughly.

File: c3po/job/utils/facebook.py (hunk 1 above)
Comment (thescriptninja) [reply c3po/job/utils/facebook.py:52]: Ah ok :+1:

**Pull request merged**